src/components/Drawer/drawer.module.scss

```diff
@@ -15,17 +15,24 @@ $handle_height: 40px;
   }
 
   &:not(.modeless) {
-    background-color: $all-backdrops;
+    pointer-events: none;
```

Comment:
do children inherit this? ideally not otherwise we'll be fighting it in implementation. though honestly i don't know what modeless is.

Reply:
No children don't inherit this. modeless is no overlay.